### DISCUSSION TITLE
Bridge Filter by Label

### DISCUSSION THREAD

**Discussion by ghost** [keptn/keptn#4562] [Ideas]
Can we implement a new filter option in the bridge to filter by labels?

In this case I'd like to see all evaluations where `testType: full`

![image](https://user-images.githubusercontent.com/13639658/124072843-b0b32200-da84-11eb-8245-51add2ce3173.png)

**Comment by agrimmer** [comment]
Sounds reasonable for large environments. 
@johannes-b What is your opinion?

**Reply by johannes-b** [reply]
@grabnerandi had the same request given the following use-case:

Labels could be `triggeredBy:Andi`, `chaosExperiment:disconnectDatabase`
* User wants to know "How many sequences did Andi execute?" "How many `disconnectDatabase` experiments did we run last week?"
 -> this would be great for both the sequence screen as well as the API
